src/openms_gui/source/VISUAL/Spectrum1DWidget.cpp

```diff
@@ -325,13 +325,16 @@ namespace OpenMS
 
   void Spectrum1DWidget::saveAsImage()
   {
-    QString file_name = QFileDialog::getSaveFileName(this, "Save File", "", "Raster images *.bmp *.png *.jpg *.gif (*.bmp *.png *.jpg *.gif);;Vector images *.svg (*.svg)");
+    QString filter = "Raster images *.bmp *.png *.jpg *.gif (*.bmp *.png *.jpg *.gif);;Vector images *.svg (*.svg)";
+    QString* sel_filter = new QString;
```

Comment:
I would use 
QString sel_filter;
and pass &sel_filter so you don't need the delete later

Reply:
done.